libraries/apollo-debug-server/graphql/schema.graphqls

```diff
@@ -0,0 +1,45 @@
+schema {
+  query: Query
+}
+
+scalar Fields
+
+"""
+ The root query
+"""
+type Query {
+  apolloClients: [ApolloClient!]!
+
+  """
+   Returns null if an ApolloClient with the given id is not found.
+  """
+  apolloClient(id: ID!): ApolloClient
+}
+
+type ApolloClient {
+  id: ID!
+
+  displayName: String!
+
+  normalizedCaches: [NormalizedCache!]!
+
+  normalizedCache(id: ID!): NormalizedCache
+}
+
+type NormalizedCache {
+  id: ID!
+
+  displayName: String!
+
+  recordCount: Int!
+
+  records: [Record!]!
+}
+
+type Record {
+  key: String!
+
+  fields: Fields!
+
+  sizeInBytes: Int!
+}
```

Comment:
New `apollo-execution` can generate a SDL file from the code itself (it's now truly code-first). This file is generated from the Kotlin code and checked in source control to monitor for API changes (just like BCV but for the network boundary)

Reply:
Super neat! 👍

Reply:
Now that I think about it, there should be a "schemaDump" and "schemaCheck" task, just like BCV too, I'll think of something